fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchNumberTest.java

```diff
@@ -274,19 +294,77 @@ public void testSearchNumber_decimal_or() throws Exception {
         assertSearchReturnsComposition("subject:Basic.decimal", "1,le100");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ge99.99");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ge99");
-        
+
         assertSearchReturnsComposition("subject:Basic.decimal", "ne13,1");
         assertSearchReturnsComposition("subject:Basic.decimal", "ap99.99,1");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ne13");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ap99.99");
     }
-    
-//    @Test
-//    public void testSearchNumber_decimal_chained_missing() throws Exception {
-//        assertSearchReturnsComposition("subject:Basic.decimal:missing", "false");
-//        assertSearchDoesntReturnComposition("subject:Basic.decimal:missing", "true");
-//        
-//        assertSearchReturnsComposition("subject:Basic.missing-decimal:missing", "true");
-//        assertSearchDoesntReturnComposition("subject:Basic.missing-decimal:missing", "false");
-//    }
-}
+
+    /*
+     * <PRE>
+     * {
+     * "url": "http://example.org/precision",
+     * "valueDecimal": 100.25
+     * }
+     * </PRE>
+     */
+    @Test
+    public void testSearchNumber_Precision() throws Exception {
+        // [parameter]=100
+        // Value: 100
+        // Precision: 3 
+        // Implied Range: [99.5 ... 100.5)
+        // Target: 100.25
+        assertSearchReturnsSavedResource("precision", "100");
+        assertSearchDoesntReturnSavedResource("precision", "101");
+        assertSearchDoesntReturnSavedResource("precision", "99");
+        assertSearchDoesntReturnSavedResource("precision", "99.99");
+
+        // [parameter]=100.00
+        // Value: 100.00
+        // Precision: 5
+        // Implied Range: [99.995 ... 100.005)
+        assertSearchReturnsSavedResource("precision", "100");
+        assertSearchDoesntReturnSavedResource("precision", "100.005");
+        assertSearchDoesntReturnSavedResource("precision", "99");
+        assertSearchDoesntReturnSavedResource("precision", "99.999999990");
+
+        // [parameter]=1e2
+        // Value: 1e2
+        // Precision: 1
+        // Implied Range: [95 ... 105)
+        assertSearchReturnsSavedResource("precision", "1e2");
+        assertSearchDoesntReturnSavedResource("precision", "1.05e2");
+
+        // [parameter]=1e3
+        // Value: 1e3
+        // Precision: 0
+        // Implied Range: [50 ... 150)
+        assertSearchReturnsSavedResource("precision", "100");
```

Comment:
```suggestion
        assertSearchDoesntReturnSavedResource("precision", "1e3");
```

Reply:
This is a good find.  1e3 wasn't the intent of the test.  
I've fixed the test (the range was right). 

Fix is coming.